Shared/View/CreateActivityView.swift

```diff
@@ -49,6 +52,17 @@ struct CreateActivityView: View {
                 }
                 TextField("Name", text: $newFriend) {
                     self.friendList.append(self.newFriend)
+                    Task {
+                        do {
+                            newFriendID = try await UserViewModel.user(named: self.newFriend)?.id ?? ""
+                            if !newFriendID.isEmpty {
```

Comment:
Should check for nil directly instead of defaulting to empty string then check for empty.